README.md

```diff
@@ -135,6 +135,56 @@ signingConfigs {
 }
 ```
 
++**Store sensitive data (API keys & urls, tracking ids, etc) in build.gradle & gradle.properties files** instead of config.xml file that can be easily decompiled. Otherwise your data is compromised.
+Additional advantage is that you can access those constants with BuildConfig.GOOGLE_API_KEY where Context is not available. 
+
+First, add constants into a `gradle.properties` file which should _not_ be added to the version control system:
```

Comment:
`_not_` looks strange, maybe capitalizing letters or bolding would be enough ;-)